src/main/java/ladder/domain/LadderSize.java

```diff
@@ -0,0 +1,43 @@
+package ladder.domain;
+
+import ladder.exception.InvalidLadderHeightException;
+import ladder.exception.InvalidParticipantsCountException;
+
+public class LadderSize {
+    private static final int MIN_PARTICIPANT_COUNT = 1;
+    private static final int MIN_LADDER_HEIGHT = 1;
+    private final int countOfParticipants;
+    private final int ladderHeight;
```

Comment:
멤버변수와 상수는 개행으로 영역을 구분해주세요!
```suggestion
    private static final int MIN_PARTICIPANT_COUNT = 1;
    private static final int MIN_LADDER_HEIGHT = 1;
    
    private final int countOfParticipants;
    private final int ladderHeight;
```

Reply:
아 넵 실수했네요!